src/main/java/org/apache/commons/lang3/CharSetUtils.java

```diff
@@ -45,8 +45,8 @@ public class CharSetUtils {
      * </pre>
      *
      * @see CharSet#getInstance(String...) for set-syntax.
-     * @param str  String to look for characters in, may be null
-     * @param set  String[] set of characters to identify, may be null
+     * @param str  String to look for characters in, may be {@code null}
```

Comment:
Is there a reason why "String" is sometimes linked and sometimes not?

Reply:
No Reason at all.